RevenueCatUI/PaywallView.swift

```diff
@@ -129,6 +132,16 @@ private struct LoadedOfferingPaywallView: View {
         }
     }
 
+    private var shouldHidePaywall: Bool {
+        switch self.mode {
+        case .fullScreen:
+            return false
+
+        case .card, .banner:
+            return self.purchaseHandler.purchased
+        }
+    }
+
```

Comment:
@aboedo this looks pretty good for now :) and it gets animated.

Reply:
wonder if we should even consider making that variable public so that you can use it in the app